pkg/pgtune/parallel_test.go

```diff
@@ -10,65 +10,86 @@ import (
 
 // parallelSettingsMatrix stores the test cases for ParallelRecommender along
 // with the expected values for its keys
-var parallelSettingsMatrix = map[int]map[string]string{
-	2: map[string]string{
+var parallelSettingsMatrix = map[int]map[int]map[string]string{
+	2: {defaultMaxBackgroundWorkers: {
```

Comment:
think this is easier (heh) to read if on a new line

Reply:
I ran it through a beautifier, hope it's better